lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context_and_proc.h

```diff
@@ -271,6 +271,7 @@ typedef struct smf_context_s {
   session_ambr_t selected_ambr;
   teid_upf_gnb_t gtp_tunnel_id;
   paa_t pdu_address;
+  eps_subscribed_qos_profile_t subscribed_qos;
```

Comment:
Can we make use of "qos_flow_request_list_t subscribed_qos_profile" instead of the new variable.

Reply:
i guess better to have a separate variable because qos_flow_request_list_t is used to store qos flows from sessiond